### DISCUSSION TITLE
Discord Server?

### DISCUSSION THREAD

**Discussion by kayhantolga** [betalgo/openai#270] [Polls]
I'm considering creating a Discord server with a bot that can help with library-related questions. Additionally, we can share our projects and discuss ideas. Do you think this would be useful? Would you be interested?

**Comment by Frogley** [comment]
I only sign into Discord for Midjourney, sorry...

**Comment by dspivak01** [comment]
Circling back to this. Was a Discord server ever created?

**Reply by kayhantolga** [reply]
https://github.com/betalgo/openai/discussions/598
